ofctrl/fgraphMeter.go

```diff
@@ -0,0 +1,116 @@
+package ofctrl
+
+import (
+	"github.com/contiv/libOpenflow/openflow13"
+	"github.com/contiv/libOpenflow/util"
+)
+
+type MeterFlag int
+type MeterType uint16
+
+const (
+	MeterKbps  MeterFlag = 0b0001
+	MeterPktps MeterFlag = 0b0010
+	MeterBurst MeterFlag = 0b0100
+	MeterStats MeterFlag = 0b1000
+
+	MeterDrop         MeterType = 1      /* Drop packet. */
+	MeterDSCPRemark   MeterType = 2      /* Remark DSCP in the IP header. */
+	MeterExperimenter MeterType = 0xFFFF /* Experimenter meter band. */
```

Comment:
I see these constant have been defined in libOpenflow, do they need to be defined here too?

Reply:
I'm thinking that others who are using it don't need to import `libopenflow` in addition.

Reply:
The module has dependency on libOpenflow, the consumers have no choice other than importing libOpenflow. Are other constants defined redundantly too? If no, I would prefer keep them same to avoid inconsistent constant usage.

Reply:
In fgraphGroup.go, I saw it defines:
```
const (
	GroupAll GroupType = iota
	GroupSelect
	GroupIndirect
	GroupFF
)
```
which are defined in libOpenflow.

Reply:
OK, thanks for the information.